kaitai/struct_test.go

```diff
@@ -0,0 +1,48 @@
+package kaitai
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// Declare two struct simulating types generated by Kaitai Struct compiler
+type oneStruct struct {
+	one int
+	_io *Stream
+}
+
+func (s *oneStruct) Kaitai_IO() *Stream {
+	return s._io
+}
+
+type twoStruct struct {
+	two int
+	_io *Stream
+}
+
+func (s *twoStruct) Kaitai_IO() *Stream {
+	return s._io
+}
+
+func WorkWithStruct(t *testing.T, s Struct, expectedSize int) {
+	t.Helper()
+	actualSize, err := s.Kaitai_IO().Size()
+	assert.Nil(t, err)
+	assert.Equal(t, actualSize, int64(expectedSize))
```

Comment:
In reality, there's tons of other tests also in this package, and I see that they really could benefit from clarity of one-line assert, rather than just implementing ad hoc manual comparisons, which are incredibly repetitive, look clumsy and error-prone. Let's leave it, providing room for further grow.